contracts-periphery/src/test/UmbraBatchSend.t.sol

```diff
@@ -0,0 +1,157 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import "./utils/DSTestPlus.sol";
+import "../UmbraBatchSend.sol";
+
+interface UmbraToll {
+  function toll() external returns(uint256);
+}
+
+abstract contract UmbraBatchSendTest is DSTestPlus {
+
+  UmbraBatchSend router;
+  MockERC20 token;
+
+  address constant umbra = 0xFb2dc580Eed955B528407b4d36FfaFe3da685401;
+  address tokenAddr;
```

Comment:
nit, but let's move `tokenAddr` right below `token`, since they are the same value but different types. Alternatively you can delete one and just cast to the other type as needed